worker-document/src/main/java/com/hpe/caf/worker/document/scripting/JavaScriptEngine.java

```diff
@@ -24,24 +26,41 @@
 import javax.script.ScriptEngine;
 import javax.script.ScriptEngineManager;
 import javax.script.ScriptException;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.HostAccess;
 
 public final class JavaScriptEngine implements ObjectCodeProvider
 {
-    private final ScriptEngine scriptEngine;
-    private final Bindings scriptEngineBindings;
+    private final ScriptEngine graalEngine;
+    private final Bindings graalEngineBindings;
+    private final ScriptEngine nashornEngine;
+    private final Bindings nashornEngineBindings;
     private final Object scriptEngineBindingsLock;
 
     public JavaScriptEngine()
     {
-        this.scriptEngine = new ScriptEngineManager().getEngineByName("nashorn");
-        this.scriptEngineBindings = scriptEngine.getBindings(ScriptContext.ENGINE_SCOPE);
+        this.graalEngine = GraalJSScriptEngine.create(
+            null,
+            Context.newBuilder("js")
+                .allowExperimentalOptions(true) // Needed for loading from classpath
+                .allowHostAccess(HostAccess.ALL) // Allow JS access to public Java methods/members
+                .allowHostClassLookup(s -> true) // Allow JS access to public Java classes
+                .option("js.load-from-classpath", "true"));
+
+        this.graalEngineBindings = graalEngine.getBindings(ScriptContext.ENGINE_SCOPE);
+        this.nashornEngine = new ScriptEngineManager().getEngineByName("nashorn");
+        this.nashornEngineBindings = nashornEngine.createBindings();
```

Comment:
Why is this changing from `.getBindings(ScriptContext.ENGINE_SCOPE);` like it was before?